backend-tests/tests/test_rbac_v2.py

```diff
@@ -496,42 +475,66 @@ def test_deploy_to_group(self, clean_mongo, test_case):
                         "permission_sets_with_scope": [
                             {
                                 "name": "DeployToDevices",
-                                "scope": {"type": "DeviceGroups", "value": ["test"],},
-                            },
-                        ],
-                    },
+                                "scope": {"type": "DeviceGroups", "value": ["test"]}
+                            }
+                        ]
+                    }
                 ],
                 "device_groups": {"test": 1, "production": 1},
                 "deploy_group": "production",
                 "set_configuration_status_code": 403,
-                "deploy_configuration_status_code": 403,
+                "deploy_configuration_status_code": 403
             },
             {
                 "name": "Test RBAC configuration deployment forbidden pat",
                 "use_personal_access_token": True,
                 "user": {
                     "name": "test1-UUID@example.com",
                     "pwd": "password",
-                    "roles": ["test"],
+                    "roles": ["test"]
                 },
                 "roles": [
                     {
                         "name": "test",
                         "permission_sets_with_scope": [
                             {
                                 "name": "DeployToDevices",
-                                "scope": {"type": "DeviceGroups", "value": ["test"],},
+                                "scope": {"type": "DeviceGroups", "value": ["test"]}
                             },
-                            {"name": "ManageTokens",},
-                        ],
-                    },
+                            {"name": "ManageTokens"}
+                        ]
+                    }
                 ],
                 "device_groups": {"test": 1, "production": 1},
                 "deploy_group": "production",
                 "set_configuration_status_code": 403,
-                "deploy_configuration_status_code": 403,
+                "deploy_configuration_status_code": 403
             },
-        ],
+            {
+                "name": "Test RBAC configuration deployment forbidden",
+                "use_personal_access_token": False,
+                "user": {
+                    "name": "test1-UUID@example.com",
+                    "pwd": "password",
+                    "roles": ["test"],
+                },
+                "roles": [
+                    {
+                        "name": "test",
+                        "permission_sets_with_scope": [
+                            {
+                                "name": "DeployToDevices",
```

Comment:
what about checking that configuration is not accessible for users with built-in Read Access role?

Reply:
Done!